### PR TITLE
Updated JS exercise 4

### PR DESCRIPTION
**Pull request by rohan-repos**

### CODE REVIEW

File: JS_exercises/exercise-3/index.js [hunk 1]
```diff
@@ -18,3 +18,4 @@ console.log("First Name: "+firstName+" Type: "+typeof(firstName));
 console.log("courseCompleted: "+courseCompleted+" Type: "+typeof(courseCompleted));
 console.log("Final Score: "+finalScore+" Type: "+typeof(finalScore));
 console.log("Winning Numbers; "+winningNumbers+" Type: "+typeof(winningNumbers));
+let result = prompt("Enter some number here", [20]);
```
Comment (chriztie) [inline JS_exercises/exercise-3/index.js:21]: The second parameter shouldn't be an array but a single value.
`let result = prompt("Enter some number here", 20);`

**Review by chriztie — APPROVED** [review]
Good job @rohan-repos 💯 you may merge this PR and see these changes in master now 🙂

File: JS_exercises/exercise-3/index.js (hunk 1 above)
Comment (rohan-repos) [reply JS_exercises/exercise-3/index.js:21]: it did not throw me any error and it worked fine even if i added multiple values

Comment (chriztie) [reply JS_exercises/exercise-3/index.js:21]: Did you try cancelling the prompt? When you define as an array it returns an object, not a single value

**Pull request merged**